platforms/software/build-init/src/main/java/org/gradle/api/tasks/wrapper/Wrapper.java

```diff
@@ -329,9 +330,13 @@ public void setDistributionType(DistributionType distributionType) {
     /**
      * The list of available gradle distribution types.
      */
-    @ToBeReplacedByLazyProperty(comment = "Not supported yet", issue = "https://github.com/gradle/gradle/issues/29341")
+    @Deprecated
     @OptionValues("distribution-type")
```

Comment:
❓ Should we remove `OptionValues` from here as well? To make sure we're using auto-generated?